mbf_utility/src/navigation_utility.cpp

```diff
@@ -42,6 +42,8 @@
 
 namespace mbf_utility
 {
+  geometry_msgs::TwistStamped actual_vel;
+  boost::mutex vel_mutex;
```

Comment:
Just curious why no [std::mutex](https://en.cppreference.com/w/cpp/thread/mutex)?
Maybe name it `<full_name>_mutex`

Reply:
include is missing